travis_install.sh

```diff
@@ -0,0 +1,8 @@
+# installs bazel so travis can build and run tests
+mkdir tmp
+cd tmp
+curl --location --compressed https://github.com/bazelbuild/bazel/releases/download/0.5.4/bazel-0.5.4-installer-linux-x86_64.sh > bazel-0.5.2-installer-linux-x86_64.sh
```

Comment:
there's a mix of 0.5.4 and 0.5.2 strings in here
should probably be extracted as a variable

Reply:
OK.